docs/guides/systemjs.md

```diff
@@ -0,0 +1,73 @@
+# Using Enzyme with SystemJS
+
+If you are using a test runner that runs code in a browser-based environment,
+you may be using [SystemJS]() in order to bundle your React code.
+
+SystemJS uses static analysis to create a dependency graph at build-time of
+your source code to build a bundle. Enzyme has a hand full of conditional
+`require()` calls in it in order to remain compatible with React 0.13 and
+React 0.14.
+
+Unfortunately, these conditional requires mean there is a bit of extra setup
+with bundlers like SystemJS.
+
+In your SystemJS configuration, you simply need to make sure that the
+following files are labeled as "external", which means they will be ignored:
+
+```
+react/addons
```

Comment:
since down below you specify the externals for react 0.14 and 15 specifically it's unclear for which version of react this list up here is meant for